.github/workflows/doc-automation.yml

```diff
@@ -0,0 +1,45 @@
+
+# This action is based on the automation in pytorch/serve. Edit as appropriate
+# TODO: Uncomment this when the automation should run on commits to main
+#on:
+#  push:
+#    branches:
+#      - main
+# TODO: Comment or delete this whole section when automation should run on commits to main
+on: 
+  workflow_dispatch:
+jobs:
+  build_docs_job:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6]
```

Comment:
I would recommend a newer python 3.6 to prevent rework when it gets deprecated (should be the next version to be discontinued)

Reply:
This is what I got from the pytorch docs folks. I will talk to them about versioning.